source/clear-linux/guides/maintenance/autospec.rst

```diff
@@ -1,189 +1,249 @@
+===========================
 
-      cd ~/clearlinux
+This example shows the basic steps for first time setup of autospec.
 
-#. Enter the command:
+#. Download the :file:`user-setup.sh` script.
 
    .. code-block:: bash
 
-      make autospecnew URL="https://github.com/clearlinux/helloclear/archive/helloclear-v1.0.tar.gz"
-      NAME="helloclear"
+      curl -O https://raw.githubusercontent.com/clearlinux/common/master/user-setup.sh
 
-   .. note::
+#. Make :file:`user-setup.sh` executable.
 
-      For a local tarball, use this type of *URL*: \file://<absolute-path-to-tarball>
+   .. code-block:: bash
 
-#. If build failures or dependency issues occur, continue below.
-   Otherwise, skip directly to `Next steps`_.
+      chmod +x user-setup.sh
 
-   #. Navigate to the specific package.
+#. Run the script as an unprivileged user.
 
-      .. code-block:: bash
+   .. code-block:: bash
 
-         cd ~/clearlinux/packages/[package-name]
+      ./user-setup.sh
 
-   #. Respond to the build process output by editing control files to resolve
-      issues, which may include dependencies or exclusions.
-      See `autospec readme`_
+#. After the script completes, log out and log in again to complete the setup
+   process.
 
-   #. Run this command:
+#. Set your Git user email and username for the repos on your system
 
-      .. code-block:: bash
+   .. code-block:: bash
 
-         make autospec
+      git config --global user.email "you@example.com"
+      git config --global user.name "Your Name"
 
-   Repeat the last two steps above until all errors are resolved and you
-   complete a successful build.
+Example 2: Build RPM with existing spec file
+============================================
 
-**Congratulations!**
+This example shows how to build a RPM from a pre-packaged upstream package, with
+an existing spec file. The example uses the ``dmidecode`` package.
 
-You've successfully created a RPM.
+Make sure you have completed `Example 1: First time setup`_ before starting
+this example.
 
-Skip to `Next steps`_.
+#. Navigate to the autospec workspace and clone the ``dmidecode`` package.
 
-.. _build-source-code-with-existing-spec-file:
+   .. code-block:: bash
 
-Option 2: Build source code with an existing spec file
-======================================================
+      cd ~/clearlinux
+      make clone_dmidecode
 
-Use this method if you only want to build the RPM using the spec file. This
-method assumes that a spec file already exists. In this example, we run a
-``make build`` on the ``dmidecode`` package.
+   Or you can clone all package repos at once using:
 
-#. Navigate to the ``dmidecode`` package in clearlinux:
+   .. code-block:: bash
+
+      make [-j NUM] clone-packages
+
+#. Navigate to the local copy of the ``dmidecode`` package and build it:
 
    .. code-block:: bash
 
       cd ~/clearlinux/packages/dmidecode/
+      make build
+
+#. The resulting RPMs are in :file:`./rpms`. Logs are in :file:`./results`.
```

Comment:
The resulting RPMs are in :file:`./rpms`. Build logs and additional RPMs are in :file:`./results`.